charts/istio/istio-ingress/templates/deployment.yaml

```diff
@@ -127,9 +127,14 @@ spec:
           value: sni-dnat
         - name: ISTIO_META_CLUSTER_ID
           value: "Kubernetes"
+        - name: ISTIO_BOOTSTRAP_OVERRIDE
+          value: /etc/istio/custom-bootstrap/custom_bootstrap.yaml
         volumeMounts:
         - mountPath: /var/run/secrets/istio
           name: istiod-ca-cert
+        - mountPath: /etc/istio/custom-bootstrap
```

Comment:
Can you add a checksum annotation for this configmap mount?

Reply:
fixed in e71bb390e